Patches/Expanded Materials - Plastics/Items_Resource_Stuff.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Expanded Materials - Plastics</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
```

Comment:
Plastics also need their heat armor reduced to CE scaled levels probably, so that any decently thick material made of them doesn't instantly cap out heat armor.

Reply:
Addtionally, the T2 polyermer should probably not have steel weapon tag, just because it actually makes a terrible weapon with so much less pen. Maybe not the steelable armor vest tag either but I'm less sure on that.

Reply:
Removed the metallic weapon tag. However, the steeled tag is probably appropriate, as I've essentially conceptualized the T2 plastic as UHMWPE, which is being used high-strength, low-weight ballistic armor. However, UHMWPE is also a good thermal conductor, so it offers pretty poor heat armor.